lib/shared/bucketing-assembly-script/__tests__/types/dvcEvent.test.ts

```diff
@@ -0,0 +1,75 @@
+import {
+    testDVCEventClass,
+    testDVCRequestEventClass
+} from '../../build/bucketing-lib.debug'
+
+const testDVCEvent = (event: unknown): unknown => {
+    return JSON.parse(testDVCEventClass(JSON.stringify(event)))
+}
+
+const testDVCRequestEvent = (event: unknown, user_id: unknown, featureVars: unknown): unknown => {
+    return JSON.parse(testDVCRequestEventClass(
+        JSON.stringify(event),
+        user_id as string,
+        JSON.stringify(featureVars)
+    ))
+}
+
+describe('DVCEvent tests', () => {
+    describe('DVCEvent', () => {
+        it('should test DVCEvent json parsing', () => {
+            const eventObj = {
```

Comment:
should you be testing for invalid values here or something

Reply:
That is already handled in other test files.